cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -689,13 +689,9 @@ func (a *StaticAutoscaler) addUpcomingNodesToClusterSnapshot(upcomingCounts map[
 		}
 		isUpcomingNodeGroup := a.processors.AsyncNodeGroupStateChecker.IsUpcoming(nodeGroup)
 		for _, upcomingNode := range upcomingNodes {
-			var pods []*apiv1.Pod
-			for _, podInfo := range upcomingNode.Pods() {
-				pods = append(pods, podInfo.Pod)
-			}
-			err := a.ClusterSnapshot.AddNodeWithPods(upcomingNode.Node(), pods)
+			err := a.ClusterSnapshot.AddNodeInfo(upcomingNode)
```

Comment:
this would be a good opportunity to rename vars as follows:

- `upcomingNodes` --> `upcomingNodeInfos`
- `upcomingNode` --> `upcomingNodeInfo`

Reply:
Done